### PR TITLE
Send an email

### PR DESCRIPTION
**Pull request by MaaooVaass**
## Summary
Send an email

## Card
https://trello.com/c/ANn17kT0/11-mandar-email-al-usuario-cuando-se-crea-un-rent

### CODE REVIEW

File: app/controllers/rents_controller.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ def create
     rent = Rent.new(rent_params)
     if rent.save
       render json: rent, status: :created
+      RentWorker.perform_async(rent.id)
```
Comment (lmasello) [inline app/controllers/rents_controller.rb]: Call the worker before rendering the response.

File: app/workers/rent_worker.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class RentWorker
```
Comment (lmasello) [inline app/workers/rent_worker.rb]: Use a more self-descriptive name. Something that when you read it, you can understand what it does. In this case, it should be something like 'RentNotificationWorker' or so.

File: app/views/rent_mailer/rent_notification.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<h1>Rent#send</h1>
```
Comment (lmasello) [inline app/views/rent_mailer/rent_notification.html.erb:1]: This is ok but just for you to know: Usually, we use slim for the views.

File: config/environments/development.rb [hunk 4]
```diff
@@ -60,4 +60,14 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => Rails.application.secrets.user_name,
```
Comment (lmasello) [inline config/environments/development.rb]: `user_name: Rails.application.secrets.user_name`

The same for the others.

**Review by lmasello — CHANGES_REQUESTED** [review]

File: config/environments/development.rb [hunk 5]
```diff
@@ -6,11 +6,11 @@
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
-
+  
   # Do not eager load code on boot.
   config.eager_load = false
 
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  # config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (fedeest) [inline config/environments/development.rb]: Eras commented code

File: app/workers/rent_notification_worker_worker.rb [hunk 6]
```diff
@@ -1,4 +1,4 @@
-class RentWorker
+class RentNotificationWorkerWorker
```
Comment (lmasello) [inline app/workers/rent_notification_worker_worker.rb]: Careful with the end of this word

File: config/environments/development.rb [hunk 7]
```diff
@@ -41,7 +41,7 @@
   config.active_support.deprecation = :log
 
   # Raise an error on page load if there are pending migrations.
-  config.active_record.migration_error = :page_load
+  config.active_record.migration_error = :page_loadÍ
```
Comment (lmasello) [inline config/environments/development.rb]: Same here, be careful with the end.

**Review by lmasello — CHANGES_REQUESTED** [review]
Also, you have a file called 'dump.rdb'. This should not be in the repository.

File: app/controllers/rents_controller.rb [hunk 8]
```diff
@@ -3,8 +3,8 @@ class RentsController < ApplicationController
   def create
     rent = Rent.new(rent_params)
     if rent.save
+      RentNotificationWorkerWorker.perform_async(rent.id)
```
Comment (fedeest) [inline app/controllers/rents_controller.rb]: You have 'Worker' both times

File: app/workers/rent_notification_worker_worker.rb (hunk 6 above)
Comment (fedeest) [reply app/workers/rent_notification_worker_worker.rb]: You have 'worker' twice

**Review by fedeest — CHANGES_REQUESTED** [review]

**Review by fedeest — APPROVED** [review]

**Review by lmasello — APPROVED** [review]

**Pull request merged**